gdk4/src/auto/clipboard.rs

```diff
@@ -298,21 +298,6 @@ impl Clipboard {
         }))
     }
 
-    pub fn get_property_local(&self) -> bool {
```

Comment:
Like this property

Reply:
This one is found to be duplicate of [`is_local`](https://github.com/gtk-rs/gtk4-rs/blob/8107dfccc9c8a3939f1eaf45b23911dfc05d986e/gdk4/src/auto/clipboard.rs#L47). Either it's fine, or you need to use `rename` or `bypass_auto_rename` in `Gir.toml` to control what should get generated.

Reply:
Aha, Indeed. I will have a look at the other ones